web/components/reset-password.jsx

```diff
@@ -0,0 +1,90 @@
+import AlertActions from '../actions/alert-actions';
+import Formsy from 'formsy-react';
+import PageHeader from './controls/page-header.jsx';
+import React from 'react';
+import request from '../request-agent';
+import Spinner from './controls/spinner.jsx';
+import TextBox from './controls/text-box.jsx';
+
+import {
+	Button,
+	Col,
+	Grid,
+	Row
+} from 'react-bootstrap';
+
+const ALERT_KEY = 'reset-password';
+
+class ResetPassword extends React.Component {
+	constructor() {
+		super();
+		this.state = {
+			sending: false
+		};
+		this.submit = this.submit.bind(this);
+	}
+
+	submit(model, reset) {
+		AlertActions.dismissAlert(ALERT_KEY);
+		this.setState({ sending: true });
+		request
+			.get('/api/auth/resetPassword')
+			.query({
+				email: model.email
+			})
+			.then(() => {
+				AlertActions.showSuccess(
+					ALERT_KEY,
+					'Password Reset Request Sent',
+					'Check your e-mail for a message containing a link to a page where you can reset your password.');
```

Comment:
Maybe elaborate on this to say that if they don't receive an e-mail it may be because the e-mail address they specified is not registered with the app.